analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposeFromDispose.CSharp9.Part1.cs

```diff
@@ -57,30 +57,26 @@ record R2 : IDisposable
 
 public partial class ResourceHolder : IDisposable
 {
-    private FileStream fs;
-
-    public void OpenResource(string path) =>
-        this.fs = new FileStream(path, FileMode.Open);
-
-    public partial void CleanUp();
+    public partial void CleanUp() =>
+        this.fs.Dispose(); // Compliant. See also test case DisposedInDispose in CSharp7_2
 
-    public partial void Dispose();
+    public partial void Dispose() =>
+        this.fs.Dispose();
 }
 
-public partial class ResourceHolder : IDisposable
+public partial class ResourceHolder2 : IDisposable
 {
     public partial void CleanUp() =>
-        this.fs.Dispose(); // Noncompliant
+        this.fs.Dispose(); // Compliant. See also test case DisposedInDispose in CSharp7_2
 
     public partial void Dispose() =>
         this.fs.Dispose();
 }
```

Comment:
These two test cases are identical. Also there is no Noncompliant test case for partial classes.